.github/workflows/test_ipex.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        transformers-version: ["4.47.0", "4.47.1"]
-        torch-version: ["2.4.0", "2.5.*"]
+        transformers-version: ["4.47.*"]
+        torch-version: ["2.6.0"]
```

Comment:
we can keep a test with a lower torch version, both option works for me

Reply:
I think there is no need to keep the previous version of pytorch tests, as the ipex tests take too much time which might be hanged if too many tasks run together.

Reply:
ok works for me